components/brave_wallet/browser/internal/cardano_tx_decoder.rs

```diff
@@ -518,7 +580,7 @@ fn extract_inputs(
         };
 
         // Extract transaction hash and index
-        let tx_hash: [u8; 32] = match &input_array[0] {
+        let tx_hash: [u8; CARDANO_TX_HASH_SIZE] = match &input_array[0] {
```

Comment:
This file is kind of inconsistent in how it uses magic constants vs named constants.

In general, I ironically find that the style recommendation to avoid magic constants makes code harder to read. Because what we do with wallet is all based in fixed-size hashes that are well-documented at a protocol level, I find the SCREAMING_CASE names to make code harder to follow.

I'm not sure there's anything actionable here but I was reading this file and noticed we were putting what the magic number was in comments, but then in this diff we replace the magic number with a name so there's inconsistency here.

Reply:
```
error[cxxbridge]: unsupported expression, array length must be an integer literal
```
Oh!

Also, ugh. So we can't use named constants in cxxbridge code? That's good to know. That's also pretty lame as well, ha ha.

Alright, I think we can consider this resolved then.